pkg/services/validationService.go

```diff
@@ -88,28 +88,40 @@ func (vs *ValidationService) Validate(admissionReviewReq *admission.AdmissionRev
 	ciContext := ciContext.Extract()
 
 	clusterK8sVersion := getK8sVersion()
+
+	rootObject := getResourceRootObject(admissionReviewReq)
+	namespace, resourceKind, resourceName, managers := getResourceMetadata(admissionReviewReq, rootObject)
+
+	clientId := getClientId()
+	policyName := os.Getenv(enums.Policy)
+
 	token, err := getToken(cliServiceClient)
 	if err != nil {
 		panic(err)
 	}
 
-	rootObject := getResourceRootObject(admissionReviewReq)
-	namespace, resourceKind, resourceName, managers := getResourceMetadata(admissionReviewReq, rootObject)
-
 	// intercept datree configuration requests
 	if config.IsConfigurationChangeEvent(admissionReviewReq.Request) {
 		vs.configHandler.HandleConfigurationChange(admissionReviewReq.Request)
 	}
 
+	// no token provided - skip validation
+	if token == "<DATREE_TOKEN>" {
+		clusterRequestMetadata := getClusterRequestMetadata(cliEvaluationId, token, false, true, resourceKind, resourceName, managers, clusterK8sVersion, policyName, namespace, server.ConfigMapScanningFilters)
+		saveRequestMetadataLogInAggregator(clusterRequestMetadata)
+
+		const warningMessage1 = `⚠️  Datree is installed, but not activated since you didn't set your token.`
```

Comment:
I think it's better to have const for the messages, perhaps with a function that wraps everything, or maybe simply create an array with the values and pass it.

Nevertheless, I don't think it's super important, so it's up to you :)